src/App.tsx

```diff
@@ -8,7 +8,51 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+
 export const App: React.FC = () => {
+  const [selectTodo, setSelectTodo] = useState<Todo | null>(null);
+  const [selectOption, setSelectOption] = useState('all');
+  const [searchInput, setSearchInput] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    getTodos()
+      .then(fetchedTodos => {
+        setTodos(fetchedTodos);
+      })
+      .catch(error => {
+        // eslint-disable-next-line no-console
+        console.error('Error fetching todos:', error);
```

Comment:
Consider providing user feedback in the UI when an error occurs during the fetching of todos. Currently, the error is only logged to the console, which is not visible to users.